### DISCUSSION TITLE
Figma Tokens Pro - any details?

### DISCUSSION THREAD

**Discussion by prodkt** [six7/figma-tokens#777] [General]
We've explored and not yet found any mention of it but do apologize in advance if its been announced.

We've noticed Pro version documentation on the Figma Tokens Docs site released but aren't finding any information on the Pro Version and how to attain it. Any estimation when this will be available to the public?

**Comment by six7** [comment]
👋 I'll be updating docs and plugin community page over the weekend, you can find most details on figmatokens.com

Here's the video of me introducing Pro at a recent conference: https://m.youtube.com/watch?v=KgUG_xDXg4I
